src/main.rs

```diff
@@ -31,10 +31,13 @@ use yew::prelude::*;
 use yew::{html, Html, Properties};
 use yew_hooks::prelude::*;
 
+//use gloo_console::log;
+
 // To load in the Fibonacci example, uncomment the CONTENT and fib_model lines
 // and comment the code, language, and text_model lines. IMPORTANT:
 // rename fib_model to text_model to have it work.
-const CONTENT: &str = include_str!("../static/assembly_examples/fibonacci.asm");
+// const CONTENT: &str = include_str!("../static/assembly_examples/fibonacci.asm");
+const CONTENT: &str = include_str!("../static/assembly_examples/riscv_test.asm");
```

Comment:
Could we switch this back to the original MIPS fibonacci until we have a dropdown to switch between architectures?